### PR TITLE
TRLST-153 candidate fix

### PR DESCRIPTION
**Pull request by jwgulley**
Another Javascript-based fix, prototyped with boilerplate code - optimised / developed / documented to be production ready

### CODE REVIEW

File: trade_remedies_caseworker/templates/organisations/organisations.html [hunk 1]
```diff
@@ -5,7 +5,32 @@
...
-<table clas="sortable" data-attach="TableSort">
+<!-- Based on code found at: https://www.w3schools.com/howto/howto_js_filter_table.asp-->
+<script>
+  function searchForOrganisation() {
+    var input, filter, table, tr, td, i, txtValue;
+    input = document.getElementById("organisationSearchInput");
+    filter = input.value.toUpperCase();
+    table = document.getElementById("organisationTable");
+    tr = table.getElementsByTagName("tr");
+    console.log( filter );
```
Comment (rossmiller) [inline trade_remedies_caseworker/templates/organisations/organisations.html:16]: Can we remove the console.log please?

**Review by rossmiller — CHANGES_REQUESTED** [review]
Minor update required.

File: trade_remedies_caseworker/templates/organisations/organisations.html (hunk 1 above)
Comment (jwgulley) [reply trade_remedies_caseworker/templates/organisations/organisations.html]: Thanks - good spot.  Updated further to comment.

**Review by rossmiller — APPROVED** [review]
LGTM!

**Pull request merged**